File: SDKs/keycloak.local/1/python/src/sdk/models/operations/post_realm_users_id_logout.py
from dataclasses import dataclass, field



@dataclass
class PostRealmUsersIDLogoutPathParams:
    id: str = field(metadata={'path_param': { 'field_name': 'id', 'style': 'simple', 'explode': False }})
    realm: str = field(metadata={'path_param': { 'field_name': 'realm', 'style': 'simple', 'explode': False }})
    

@dataclass
class PostRealmUsersIDLogoutRequest:
    path_params: PostRealmUsersIDLogoutPathParams = field()
    

@dataclass
class PostRealmUsersIDLogoutResponse:
    content_type: str = field()
    status_code: int = field()
    

File: SDKs/keycloak.local/1/typescript/src/sdk/models/operations/postrealmusersidlogout.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class PostRealmUsersIdLogoutPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=realm" })
  realm: string;
}


export class PostRealmUsersIdLogoutRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: PostRealmUsersIdLogoutPathParams;
}


export class PostRealmUsersIdLogoutResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
